bridge/ffi/src/utils.rs

```diff
@@ -1,18 +1,107 @@
 use crate::bindings::*;
-use dsnp_graph_config::DsnpVersion;
+use dsnp_graph_config::{DsnpVersion, MAINNET_CONFIG, ROCOCO_CONFIG};
 use std::{collections::HashMap, mem::ManuallyDrop};
 
+// Function to convert Rust `Config` to C-compatible `Config`
+pub fn get_config_for_ffi(environment: &Environment) -> Config {
+	match environment {
+		Environment::Mainnet => {
+			let rust_config = &MAINNET_CONFIG;
```

Comment:
I think Mainnet and Rococo basically has the same logic , You can combine these by using
`environment.get_config()
`